tests/testcases/Group10-Vulnerability/10-02-Clair-scan-a-image-with-vulnerability.md

```diff
@@ -0,0 +1,26 @@
+Test 10-02  Clair scan a image with vulnerability
+=======
+  
+# Purpose:  
+To verify that clair can scan a image with vulnerability.  
+
+# References:  
+User guide  
+  
+# Environment:  
+* This test requires that a Harbor instance is running and available.  
+* Harbor is installed with clair enable.  
+* A linux host with Docker CLI installed.  
+  
+# Test Step  
+1. Login Harbor.  
+2. Push a normal image.  
+3. Scan the image.  
+4. Click tag link enter detail page.  
+  
+# Expected Outcome:  
+* Step3 should exist result summary chart.  
+* Step4 should see cve link.  
```

Comment:
Should verify:
1)The summary matches between the summary and details.
2)CVE link is clickable and point to the correct location on the internet.
3)Try to rescan, and what happens?
4)Does the color bar reflect the correct ratio of the vulnerabilities?

Some error case, shut down registry and scan, what will happen? What logs should be seen?
start registry and rescan what will happen?